app/src/main/java/com/example/coronadiagnosticapp/ui/activities/MainActivity.kt

```diff
@@ -26,6 +27,12 @@ private val REQUIRED_PERMISSIONS = arrayOf(
 class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
+        val res: Resources = this.resources
+        val dm: DisplayMetrics = res.displayMetrics
+        val conf: Configuration = res.configuration
+//        conf.setLocale(Locale("iw")) // API 17+ only.
```

Comment:
?

Reply:
I added this so we will be able to easily switch between the languages of the app